runtimes/peregrine/src/lib.rs

```diff
@@ -851,6 +850,8 @@ impl InstanceFilter<RuntimeCall> for ProxyType {
 							| did::Call::submit_did_call { .. }
 							| did::Call::update_deposit { .. }
 							| did::Call::change_deposit_owner { .. }
+							| did::Call::create_from_account { .. }
+							| did::Call::dispatch_as { .. }
```

Comment:
Seems like we forgot about these two new extrinsics ._.

Reply:
Yeah, I think we need to somehow factor them out in a way that we update it once, and changes are reflected in other places. We tend to forget the proxy stuff quite often.